### DISCUSSION TITLE
Blog post body with an embed only showing on refresh

### DISCUSSION THREAD

**Discussion by justinfriebel** [vercel/next.js#15179] [Help]
I have blog post content coming from prismic. I have one post with a GitHub gist embedded. When my site is using the cached SSG? json content it doesn't seem to load the embed even though it's there as shown in the image below. When I refresh the page the embed shows up. Any ideas on what's going on here?

https://share.getcloudapp.com/d5uW584z

**Comment by surjithctly** [comment]
Have the same issue. How did you solve?

**Reply by justinfriebel** [reply]
I'm not exactly sure because it was so long ago, but maybe the answer is hidden in my blog's repo: https://github.com/justinfriebel/nextjs-prismic-blog

**Reply by surjithctly** [reply]
I think you do not have embed anymore.
